src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -51,6 +51,30 @@ public void verifyCategoryTransitionThroughDropdownTest(String title) {
                 .contains(title);
     }
 
+    @DataProvider
+    public Object[][] rozetkaCityData() {
+        return new Object[][]{
+                {"Харків"},
+                {"Дніпро"}
+        };
+    }
+    @Test(dataProvider = "rozetkaCityData")
+    public void cityChangingTest(String expectedCityName) {
```

Comment:
verifyCityChangingTest

Reply:
also probably this test related more to Side Menu functionality than to the home page